### DISCUSSION TITLE
Difficulty to build project

### DISCUSSION THREAD

**Discussion by amit2010verma** [osmandapp/OsmAnd#11618] [Q&A]
There is too difficulty to build project. I have remove some dependency but still giving-

Out of memory: Java heap space. Configure Gradle memory settings using '-Xmx' JVM option (e.g. '-Xmx2048m'.)

Please fix the project's Gradle settings.

I have also change Gradle setting but its not working.

**Comment by bbigras** [comment]
I have the same problem.

**Comment by Zahnstocher** [comment]
`org.gradle.jvmargs=-Xmx2048m`
